### PR TITLE
Implement task CardID

### PR DESCRIPTION
**Pull request by adam-dej**
This task reads IDs of RFID cards

### CODE REVIEW

File: src/tasks/cardid-task/cardid-task.c [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                resetRFIDModule();
+                chMtxUnlock(&pollMutex);
+            } else {
+                readerActive = pollThisCycle;
+            }
+        }
+
+        if (pollThisCycle) {
+            unsigned int result_p = 0;
+            Picc cards[10];
```
Comment (lubo) [inline src/tasks/cardid-task/cardid-task.c]: I see there's a maximum of 10 cards per poll. This number is used in multiple places, so defining the constant would be appropriate.

File: src/tasks/cardid-task/cardid-task.h [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+/* Task data structures and constants                                        */
+/*===========================================================================*/
+
+/**
+ * @brief       Card ID structure
+ *
+ * This structure carries the Card ID and information about how long the ID is.
+ */
+typedef struct {
+    uint8_t     uid[10];
```
Comment (lubo) [inline src/tasks/cardid-task/cardid-task.h]: Again, defining the number as a constant would be great.

**Review by lubo — CHANGES_REQUESTED** [review]

File: src/tasks/cardid-task/cardid-task.c (hunk 1 above)
Comment (adam-dej) [reply src/tasks/cardid-task/cardid-task.c]: Fixed

File: src/tasks/cardid-task/cardid-task.h (hunk 2 above)
Comment (adam-dej) [reply src/tasks/cardid-task/cardid-task.h]: Done

File: src/tasks/cardid-task/cardid-task.c [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            } else {
+                readerActive = pollThisCycle;
+            }
+        }
+
+        if (pollThisCycle) {
+            unsigned int result_p = 0;
+            Picc cards[MAX_CARDS_PER_POLL_CYCLE];
+            bool is_that_all;
+            result_p = iso14443FindCards(PCD, cards, 10, &is_that_all);
```
Comment (lubo) [inline src/tasks/cardid-task/cardid-task.c]: Seems like you forgot to change this `10` to `MAX_CARDS_PER_POLL_CYCLE`.

**Review by lubo — CHANGES_REQUESTED** [review]

File: src/tasks/cardid-task/cardid-task.c (hunk 3 above)
Comment (adam-dej) [reply src/tasks/cardid-task/cardid-task.c]: Right, fixed.

**Review by lubo — APPROVED** [review]

**Pull request merged**